### ISSUE TITLE
Bug: Missing images in posts

### ISSUE THREAD

**Issue by maia-s** [Dimillian/IceCubesApp#1618]
## Environment:

 - OS: iOS 17.0.3
 
 - IceCubesApp version: 1.8.5
 

## Description

For some posts, Ice Cubes won't display images at all. There's no error or any other indication that an image is supposed to be there.

For example, this post demonstrates the issue: https://mastodon.world/@Thoogah/111211707822842407
None of the images from that account will show up in Ice Cubes. I've seen the problem with other accounts from that server as well.

In the web interface, images show up as normal.

## Related Issues

 - [x] Search that this bugs don't already exist before creating it.

**Comment by maia-s** [comment]
Another example: https://mastodon.world/@exocomics/111236154385135581

It seems like this affects many (all?) accounts on mastodon.world

**Comment by Shanesan** [comment]
So I have added mastodon.world to my local timelines and all images are coming through just fine.

For exocomics, the *preview* comes up fine, but I can't load the image. The error is "original data <filesize>kb". Is [this what you see](https://github.com/Dimillian/IceCubesApp/issues/1430#issuecomment-1631069873)?

**Comment by maia-s** [comment]
No, there’s no image or error. There’s no indication that anything’s supposed to be there (middle post)

![IMG_5615](https://github.com/Dimillian/IceCubesApp/assets/66437537/47102bd5-e9f5-4ae6-bd35-67e8a4e443ea)

**Comment by maia-s** [comment]
In the web interface, it says preview not available, but the image displays when i tap it

![IMG_5617](https://github.com/Dimillian/IceCubesApp/assets/66437537/1563e06b-7eb7-4489-ab4f-2dc90d7fa917)

**Comment by maia-s** [comment]
Another one. This one has a tag for the image description, which works, but no image. I tried tapping around the tag to see if it’d open the image, but that didn’t do anything. 

![IMG_5621](https://github.com/Dimillian/IceCubesApp/assets/66437537/14099fd7-f9de-4c89-9ef9-f1d63e4a6e87)

**Comment by maia-s** [comment]
This is still an issue in Ice Cubes 1.9.9

**Comment by maia-s** [comment]
Still happens in 1.9.18
